src/gainmap.c

```diff
@@ -55,7 +55,7 @@ avifBool avifGainMapMetadataFractionsToDouble(avifGainMapMetadataDouble * dst, c
 
 static void avifGainMapMetadataSetDefaults(avifGainMapMetadataDouble * metadata)
 {
-    memset(metadata, 0, sizeof(avifGainMapMetadata));
+    memset(metadata, 0, sizeof(*metadata));
```

Comment:
The function name can begin with "avifGainMapMetadataDouble" to follow the naming convention,.